roles/ampliseq/defaults/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+ampliseq_repo: "https://github.com/nf-core/ampliseq.git"
+ampliseq_dest: "{{ sw_path }}/ampliseq"
+ampliseq_container_url: "docker://nfcore/ampliseq:{{ ampliseq_version }}"
+ampliseq_container_path: "{{ ngi_containers }}/ampliseq"
+ampliseq_image: "ampliseq-{{ ampliseq_version }}.simg"
+ampliseq_version: "1.0.0"
```

Comment:
I shoud have told you earlier, but I just checked and there's a version 1.1.0 release available now. Let's go for that one instead?